lib/user.js

```diff
@@ -5,9 +5,14 @@ async function getOrganization() {
   return apiResponse;
 }
 
-async function getProfile(account_name) {
-  const apiResponse = await apiRequest('POST', '/user/profile', {}, { account_name });
+async function getProfile() {
+  const apiResponse = await apiRequest('GET', '/user/profile', {}, null);
   return apiResponse;
 }
 
-module.exports = { getOrganization, getProfile };
+async function searchUser(username) {
```

Comment:
The search is not only by username, so better name the param `username` instead to something like `query` as it lets the user search by part of names, phone, email, account name, etc